src/main/java/org/osc/controller/nsc/api/NeutronSecurityControllerApi.java

```diff
@@ -16,27 +16,23 @@
  *******************************************************************************/
 package org.osc.controller.nsc.api;
 
+import com.google.common.collect.ImmutableMap;
 import org.apache.log4j.Logger;
-import org.jclouds.openstack.neutron.v2.NeutronApi;
-import org.jclouds.openstack.neutron.v2.domain.Port;
-import org.jclouds.openstack.neutron.v2.domain.Port.UpdatePort;
-import org.jclouds.openstack.neutron.v2.features.PortApi;
-import org.osc.controller.nsc.api.jcloud.BaseJCloudApi;
-import org.osc.controller.nsc.api.jcloud.Endpoint;
-import org.osc.controller.nsc.api.jcloud.JCloudUtil;
+import org.openstack4j.model.network.Port;
+import org.osc.controller.nsc.api.openstack4j.BaseOpenstack4jApi;
+import org.osc.controller.nsc.api.openstack4j.Endpoint;
 import org.osc.controller.nsc.model.InspectionHook;
 import org.osc.controller.nsc.model.InspectionPort;
 import org.osc.sdk.controller.DefaultNetworkPort;
 import org.osc.sdk.controller.element.InspectionPortElement;
 import org.osc.sdk.controller.element.NetworkElement;
 import org.osc.sdk.controller.exception.NetworkPortNotFoundException;
 
-import java.io.Closeable;
 import java.util.Arrays;
 import java.util.List;
 import java.util.UUID;
 
-public class NeutronSecurityControllerApi extends BaseJCloudApi {
+public class NeutronSecurityControllerApi extends BaseOpenstack4jApi {
```

Comment:
@derekwb @wardtimx Hi ! Could you take a look at that sdn-controller-nsc-plugin? I succesufully managed to add only openstack4j deps(pom.xml) but i do have problem with this osgi-integration test. @arvindn05 also suggested to pack it using bnd.bnd file, but wouldn't it be too complicated? is there any possibility to "cut" imports from deps in that test(test is failing because OSGI cannot import org.openstack4j.core.transport.internal, but that package is inside openstack4j-core, whicch is imported)